lib/Doctrine/DBAL/Schema/MySqlSchemaManager.php

```diff
@@ -280,7 +280,7 @@ protected function _getPortableTableForeignKeysList($tableForeignKeys)
         return $result;
     }
 
-    public function listTableDetails($tableName)
+    public function listTableDetails($tableName) : Table
```

Comment:
If master is going to be DBAL 2.10, adding a return type would be a BC break.

Reply:
Indeed! Solved.